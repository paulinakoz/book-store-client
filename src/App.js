

function App() {
  return (
    <div className="App">
      My first component
    </div>
  );
}

export default App;
