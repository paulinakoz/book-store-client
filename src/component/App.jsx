import React from "react";
import axios from "axios";
import baseUrl from "../config/index.js";
import Layout from "./layout/Layout.jsx";
import BookContainer from "./book/BookContainer.jsx";

function App() {
  axios(`${baseUrl}/api/v1/books`).then((books) => {
    console.log(books);
  });

  return (
    <Layout>
      <BookContainer />
    </Layout>
  );
}

export default App;
