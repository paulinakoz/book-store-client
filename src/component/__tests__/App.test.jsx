import React from "react";
import renderWithRedux from "../../util/testUtil";
import App from "../App";

describe("App component", () => {
  it("should render app without error", () => {
    const { asFragment } = renderWithRedux(<App />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
