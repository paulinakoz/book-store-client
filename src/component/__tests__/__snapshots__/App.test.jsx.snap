// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render app without error 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-0"
  >
    <div
      class="MuiBox-root css-0"
    >
      <header
        class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionFixed mui-fixed css-11b3ww9-MuiPaper-root-MuiAppBar-root"
      >
        <div
          class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-dense css-10ltm04-MuiToolbar-root"
        >
          <h6
            class="MuiTypography-root MuiTypography-h6 css-1qp5wg9-MuiTypography-root"
          >
            Book Store
          </h6>
        </div>
      </header>
    </div>
    <div
      class="MuiBox-root css-1afphko"
    >
      <div
        class="MuiBox-root css-ksvmh9"
      >
        <div
          class="MuiBox-root css-1carym4"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-thsn50-MuiPaper-root"
          >
            book filter
          </div>
        </div>
        <div
          class="MuiBox-root css-1x97c6v"
        >
          Display all books.
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
